smart_proxy_discovery.gemspec

```diff
@@ -18,5 +18,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency('mocha')
   s.add_development_dependency('webmock')
 
+  s.add_runtime_dependency('mime-types', ['= 1.25.1'])
```

Comment:
Yeah, I agree. Yet it still fails (probably because rest-client is requiring the newer version of mime-types.
I think I'm gonna remove this and pin rest-client.
Thanks.